bin/stat_engine.ml

```diff
@@ -100,7 +100,10 @@ let rec sequencer_main_func num_beats tones device bpm _ =
           let note = Play.event_to_note tones event in
           Midi.(
             write_output device
-              [ message_on ~note ~timestamp:0l (* ~volume:'\070' *) () ]);
+              [
+                message_on ~note:note.Scale.note ~timestamp:0l
```

Comment:
Same as above: get `volume` and `note` by using a nice let bind statement would make it more readable. 😊